aeron-driver/src/main/c/aeron_ipc_publication.c

```diff
@@ -203,14 +203,21 @@ int aeron_ipc_publication_create(
     _pub->unblock_timeout_ns = (int64_t)context->publication_unblock_timeout_ns;
     _pub->untethered_window_limit_timeout_ns = (int64_t)params->untethered_window_limit_timeout_ns;
     _pub->untethered_resting_timeout_ns = (int64_t)params->untethered_resting_timeout_ns;
+    _pub->liveness_timeout_ns = (int64_t)context->image_liveness_timeout_ns;
     _pub->is_exclusive = is_exclusive;
+    _pub->in_cool_down = false;
+    _pub->cool_down_expire_time_ns = 0;
 
     _pub->conductor_fields.consumer_position = aeron_ipc_publication_producer_position(_pub);
     _pub->conductor_fields.last_consumer_position = _pub->conductor_fields.consumer_position;
     _pub->conductor_fields.clean_position = _pub->conductor_fields.consumer_position;
 
     _pub->unblocked_publications_counter = aeron_system_counter_addr(
         system_counters, AERON_SYSTEM_COUNTER_UNBLOCKED_PUBLICATIONS);
+    _pub->error_messages_received_counter = aeron_system_counter_addr(
```

Comment:
I think we should have a separate counter for IPC rejections.  There are no actual error frames involved here, but we would still like some observability.